pkg/auth/authproviders/openshift/internal/dexconnector/openshift.go

```diff
@@ -17,7 +17,9 @@ import (
 	"github.com/dexidp/dex/connector"
 	"github.com/dexidp/dex/storage/kubernetes/k8sapi"
 	"github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/grpc/requestinfo"
 	"github.com/stackrox/rox/pkg/httputil/proxy"
+	"github.com/stackrox/rox/pkg/netutil"
 	"github.com/stackrox/rox/pkg/set"
 	"github.com/stackrox/rox/pkg/utils"
 	"golang.org/x/oauth2"
```

Comment:
There is a table below saying what has been changed from the original file. Could you please update it with the following:
*  deduce redirect URI's hostname via `MakeRedirectURI()` function